os/android/iahwc2.cpp

```diff
@@ -344,6 +344,7 @@ HWC2::Error IAHWC2::RegisterCallback(int32_t descriptor,
       for (size_t i = 0; i < size; ++i) {
         IAHWC2::HwcDisplay *display = extended_displays_.at(i).get();
         display->RegisterVsyncCallback(data, function);
+        ETRACE("######Register original Vsync callback");
```

Comment:
If this aims to check in, do not use etrace for log only info.

Reply:
just for verifying, I will commit another one once this one work